### ISSUE TITLE
All former issues gone and no history any longer

### ISSUE THREAD

**Issue by kwin** [AdobeDocs/experience-manager-cloud-service.en#27]
All previously created issues are gone and also there is no git history any longer. What happened?

**Comment by bohnertchris** [comment]
A fair question, @kwin. During maintenance of our GitHub mirror of the AEMaaCS documentation repository, the PRs, issues, and git history were inadvertently wiped out. The issues and PRs themselves are copied to our Jira system and were not lost. However the communication threads are only maintained in GitHub, so we unfortunately had no way to recreate them on GitHub or to communicate to our readers who submitted them that they were deleted from the repo.

We are working with our documentation platform team, Experience League, to ensure that this does not happen again and we apologize for the situation.

**Comment by kwin** [comment]
@bohnertchris Thanks for the information, that is very sad. Please consider using something like https://docs.github.com/en/repositories/archiving-a-github-repository/backing-up-a-repository for keeping a proper backup of this repo.
What about the history in this repo? Seems you currently always force push only a single commit...

**Comment by kwin** [comment]
Can you please restore the PRs from your backup (I am wondering how you have them maintained in your Adobe JIRA, though as that is only for issue tracking but no actual code...)

**Comment by bohnertchris** [comment]
I agree, @kwin, that this is a very unfortunate situation and I assure you we on the AEM Docs team are very upset that this occurred and are working with the Experience League team, the maintainer of our documentation platform, to prevent this from occurring again.

With regards to the history, at any given time, we only mirror to GitHub the current state of the main branch for any of our documentation repositories. This is to keep confidential any documentation that we are preparing on other branches. For this reason, we have an automation that copies issues reported on GitHub to our Jira so we can track those tasks. However only the issue title description are copied. With issue descriptions in Jira, we have enough information to work on the tasks. However, it is not enough information to recreate the issues on GitHub due to the missing communication thread.

I regret that the Experience League team does not believe we can restore the lost issues and PRs. I am not aware of the full details of the situation, but your tip on the backup solution is appreciated and I will pass it on to them.

**Comment by aheim0** [comment]
@kwin 
Just to follow up and inform you that earlier this week the platform team performed maintenance that should prevent such events happening again. We really hope this is the case.
Thanks for your interest in the documentation.
Closing this issue now.

**Comment by kwin** [comment]
> With regards to the history, at any given time, we only mirror to GitHub the current state of the main branch for any of our documentation repositories. This is to keep confidential any documentation that we are preparing on other branches

How come that I see the history now again? Feature or Bug?

**Comment by aheim0** [comment]
Good question - I'd like to say that it's definitely a feature, but you'd be better asking the platform team.

**Comment by bohnertchris** [comment]
@kwin, I misspoke for which I apologize. Before we ran into the issue, we mirrored the entire history of the `main` branch. We did not mirror the PRs, issues, and other branches. I conflated that with also not mirroring the history, which was a mistake.

We are thankful that the platform team finally applied the fix and the repo is now once again continually mirroring the state of the `main` branch as it did before this incident occurred.
